src/sentry/tasks/post_process.py

```diff
@@ -524,7 +525,11 @@ def post_process_group(
                 )
                 return
             with metrics.timer("tasks.post_process.delete_event_cache"):
-                event_processing_store.delete_by_key(cache_key)
+                event_type = data.get("type", None)
+                if event_type == "transaction":
+                    transaction_processing_store.delete_by_key(cache_key)
+                else:
+                    event_processing_store.delete_by_key(cache_key)
```

Comment:
why are you modifying post process to start with? don't you want to complete dual-writing before cutting over reads in one go?

Reply:
`transaction_processing_store` should still point to `event_processing_store` in this first pass, so this should be ok